resources/lang/nl/worker/worker.php

```diff
@@ -10,7 +10,7 @@
 	'taxReviews' => 'Belastingoverzicht',
 	'uploadId' => 'ID Foto',
 	'uploadInstructions' => 'Upload hier een foto van je paspoort of de voor- en achterkant van je ID kaart. Let op: een rijbewijs is niet geldig als ID.',
-    'liabilityLabel' => 'Algemene voorwaarden',
+    'liabilityLabel' => 'privacy policy and privacy beleid',
```

Comment:
looks like accidentally changed translation to english

Reply:
from the email - when you fill in your data it says algemene voorwaarden but it needs to be privacy policy and privacy beleid. (See attachment)